crds/objectstorage.k8s.io_buckets.yaml

```diff
@@ -88,12 +88,19 @@ spec:
                       serviceAccount:
                         type: string
                     type: object
-                  protocolName:
-                    enum:
-                    - s3
-                    - azure
-                    - gcs
-                    type: string
+                  requestedProtocol:
+                    properties:
+                      name:
+                        enum:
+                        - s3
+                        - azure
```

Comment:
ditto on "azure" vs "azureBlob"?

Reply:
thanks for the review, changed enum to azureBlob

Reply:
thanks for the review, changed enum to azureBlob